sound/soc/intel/boards/sof_maxim_common.c

```diff
@@ -72,28 +73,61 @@ static struct snd_soc_dai_link_component max_98373_components[] = {
 	},
 };
 
+static const struct {
+	unsigned int tx;
+	unsigned int rx;
+} max_98373_tdm_mask[] = {
+	{.tx = 0x03, .rx = 0x3},
+	{.tx = 0x0c, .rx = 0x3},
+};
+
 static int max_98373_hw_params(struct snd_pcm_substream *substream,
 			       struct snd_pcm_hw_params *params)
 {
 	struct snd_soc_pcm_runtime *rtd = snd_soc_substream_to_rtd(substream);
+	struct snd_soc_dai_link *dai_link = rtd->dai_link;
 	struct snd_soc_dai *codec_dai;
-	int ret = 0;
-	int j;
+	int i, tdm_slots, ret = 0;
```

Comment:
I prefer one line per variable, and certainly initialized variables on a separate line...

Reply:
done